Wireframe/index.html

```diff
@@ -1,33 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </header>
-    <main>
-      <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
-        <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
-        </p>
-        <a href="">Read more</a>
-      </article>
-    </main>
-    <footer>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </footer>
-  </body>
+<head>
+    <meta charset="UTF-8">
+    <title>Git and Web Design Concepts</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+<header>
+    <h1>Understanding Git and Web Design Concepts</h1>
+</header>
+
+<main>
+    <article>
+        <h2>What is the Purpose of a README File?</h2>
+        <p>The README file explains what a project is, how to install and use it, and how to contribute. It serves as the main introduction and guide for anyone viewing the project.</p>
+        <a href="https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes" target="_blank">Learn more about README files</a>
+    </article>
+
+    <article>
+        <h2>What is the Purpose of a Wireframe?</h2>
+        <p>A wireframe is a visual layout that outlines the basic structure of a web page or app, showing the placement of key elements without focusing on colors or final design. It helps plan the user experience before development begins.</p>
+        <a href="https://www.adobe.com/creativecloud/design/discover/wireframe.html" target="_blank">Learn more about wireframes</a>
+    </article>
+
+    <article>
+        <h2>What is a Branch in Git?</h2>
+        <p>A Git branch allows developers to work on new features or fixes separately from the main code, ensuring that the main project stays stable. Once the work is ready, the branch can be merged back into the main codebase.</p>
+        <a href="https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell" target="_blank">Learn more about Git branches</a>
+    </article>
+</main>
+
+<footer>
```

Comment:
This footer isn't currently fixed at the bottom of the viewport like the acceptance criteria require.